components/scream/src/control/atmosphere_driver.cpp

```diff
@@ -26,29 +26,34 @@ namespace control {
  *  2) Create the grid manager, and query the atm procs for the grids they need. The GM will then
  *     proceed to build those grids (and only those grids).
  *  3) The GM is passed back to the atm procs, which can grab the needed grids, from which they can
- *     get the information needed to complete the setup of the FieldIdentifiers of their fields
- *     (both required and computed). Their field identifiers MUST be completed upon return from
+ *     get the information needed to complete the setup of the FieldRequest and GroupRequest for
```

Comment:
Do we explain in these comments what the distinction is between a field and a group?  If not, perhaps one line would make it easier to understand for any newcomers.

Reply:
Added some pointers